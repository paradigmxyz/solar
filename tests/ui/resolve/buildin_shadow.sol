//https://github.com/argotorg/solidity/blob/develop/test/libsolidity/syntaxTests/events/illegal_names_exception.sol

contract C {
	event this();
	event super();
	event _();
}