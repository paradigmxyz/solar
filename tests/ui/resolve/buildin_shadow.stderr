warning: this declaration shadows a builtin symbol
  --> ROOT/tests/ui/resolve/buildin_shadow.sol:LL:CC
   |
LL |     event this();
   |           ^^^^

warning: this declaration shadows a builtin symbol
  --> ROOT/tests/ui/resolve/buildin_shadow.sol:LL:CC
   |
LL |     event super();
   |           ^^^^^

